src/bundle.scss

```diff
@@ -0,0 +1,42 @@
+@import "./normalize";
```

Comment:
Now go to SASS documentation and read about `@import`. There's a notice about its' usage